components/python/jinja2/Makefile

```diff
@@ -11,11 +11,12 @@
 #
 # Copyright (c) 2014 Alexander Pyhalov.  All rights reserved.
 # Copyright 2015, PALO Richard.
+# Copyright 2017 Gary Mills
 #
 
 include ../../../make-rules/shared-macros.mk
 
-PATH=/usr/bin:/usr/gnu/bin:/usr/sbin
+PATH = $(PATH.ill):/usr/sbin
```

Comment:
Can we go with a more generic PATH.illumos and PATH.gnu, which would cover 90%cases? I think we could include /usr/sbin (commonly used for dtrace) and /usr/perl5/bin in both of them.